lib/chef/knife/vsphere_vm_snapshot.rb

```diff
@@ -147,17 +147,23 @@ def find_node(tree, name)
   end
 
   def display_node(node, current, shift = 1)
```

Comment:
While we're here could you change `shift` to something else? It's a method on `Array` and has other meanings, and I'd like to reduce confusion.

Reply:
Oh, now that you mention it, your improvement means we don't need `shift` any more. What it did was tell the method how much indentation to use so that child snapshots appeared "under" their parent. But you've done that by nesting it within the data structure and letting Chef figure it out, so we can do away with that parameter entirely and make the code simpler. 🎉

Reply:
ah yes that makes sense now...I originally couldn't work out what it was doing! yeap I will remove it :)